x-pack/plugins/security/server/authentication/api_keys/api_keys.ts

```diff
@@ -253,10 +260,10 @@ export class APIKeys {
   }
 
   /**
-   * Tries to invalidate an API key by using the internal user.
-   * @param params The params to invalidate an API key.
+   * Tries to invalidate an API keys by using the internal user.
```

Comment:
__question:__ would you mind updating the signature/log-messages/etc for `invalidate` method as well to keep them consistent? I see only one place in `fleet` that uses `invalidate`, so hopefully it won't require too much work. But if not, feel free to keep it as is and I'll update it in the follow-up.